Sources/Models/ListTemplateModel.swift

```diff
@@ -19,6 +19,7 @@ public struct ListTemplate: Codable {
         public let title: String?
         public let description: String?
         public let action: Action?
+        public let articleId: [String:String?]?
```

Comment:
What's the use of this? It's not on [docs](https://docs.kommunicate.io/docs/message-types#list-template).

Reply:
its not mentioned in the docs. but it was developed in android long back inseams . basically use case is if user clicks this item in list template, through events we are getting this articled object, from that user will be navigated to the app screen. lets say user taps on KFC, they will be redirected to Restaurants page in the App.

Reply:
Okay, see if it will be used by others as well, then we can list it on docs and mention it's for mobile only.

Reply:
sure will check it.